Sources/MobileSdk/MDocReaderBLEPeripheral.swift

```diff
@@ -21,6 +33,25 @@ class MDocReaderBLEPeripheral: NSObject {
     var writingQueueChunkIndex: Int?
     var writingQueue: IndexingIterator<ChunksOfCountCollection<Data>>?
 
+    var activeStream: MDocReaderBLEPeripheralConnection?
+
+    /// If this is `true`, we offer an L2CAP characteristic and set up an L2CAP stream.  If it is `false` we do neither
+    /// of these things, and use the old flow.
+    var useL2CAP = true
```

Comment:
It is; I put that there so I could manually turn it off to make sure I hadn't broken the non-L2CAP flow.

Reply:
But if you use a holder that doesn't make use of L2CAP, then it shouldn't matter?

Reply:
What happens at a high level is this:

- reader advertises (amongst other things) State, L2CAP
- if holder reads State, reader moves into original flow and L2CAP is not used
- if holder reads L2CAP instead, reader moves into L2CAP flow

The reader's `useL2CAP` boolean controls whether the reader sets up/advertises L2CAP. If it doesn't, it's not there for the holder to try to use, and only the old flow is available.
The holder's `useL2CAP` boolean controls whether the holder attempts to access L2CAP if the reader is offering it.  If this is false, the holder will always use the old flow even if L2CAP is available.
So we only get an actual L2CAP transfer if `useL2CAP` is `true` for both the reader and holder.

Reply:
In the case where a 3rd party holder doesn't support L2CAP, it will use State, which will trigger the reader to use the non-L2CAP flow.  You can simulate this by setting `useL2CAP` to `false` in our holder, and this does work properly when I test it.

Reply:
I guess it makes sense to keep it if we want to test against a wallet that we can't control whether it should use L2CAP or not